lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -1576,6 +1576,16 @@ DWARFASTParserClang::GetCPlusPlusQualifiedName(const DWARFDIE &die) {
   return qualified_name;
 }
 
+// BEGIN SWIFT  
+bool DWARFASTParserClang::IsSwiftInteropType(const DWARFDIE &die) {
+  for (DWARFDIE die : die.children())
+    if (die.Tag() == llvm::dwarf::DW_TAG_member &&
+        llvm::StringRef(die.GetName()) == "__swift_mangled_name")
```

Comment:
Regarding a conversation @felipepiovezan and I had offline about query APIs: I'm curious if this `if (die.GetName() == string)` also happens often enough to warrant a faster `die.IsName(string)` API.

Reply:
Probably worth having a `die.CompareName(other)` method too, as it is always faster to compare it that way in the negative case.